content/en/user-guide/aws/kms/index.md

```diff
@@ -136,11 +136,8 @@ This can be useful to pre-seed a development environment so values encrypted wit
 Here is an example of using custom key material with the value being base64 encoded:
 
 {{< command >}}
-$ echo 'c3VwZXIgc2VjcmV0IGtleQo=' | base64 -d
-<disable-copy>
-super secret key
-</disable-copy>
-$ awslocal kms create-key --tags '[{"TagKey":"_custom_key_material_","TagValue":"c3VwZXIgc2VjcmV0IGtleQo="}]'
+$ CUSTOM_KEY_MATERIAL=$(openssl rand -base64 32)
```

Comment:
I would rather keep the format intact and change the string to `thisisasecurekey` which would give me `dGhpc2lzYXNlY3VyZWtleQ==`.

Reply:
@sannya-singal Apologies for the delayed response. I've made the correction as you suggested.